cli/server.go

```diff
@@ -2090,7 +2090,13 @@ func IsLocalhost(host string) bool {
 	return host == "localhost" || host == "127.0.0.1" || host == "::1"
 }
 
-func ConnectToPostgres(ctx context.Context, logger slog.Logger, driver string, dbURL string) (sqlDB *sql.DB, err error) {
+// ConnectToPostgres takes a control flag "migrate". If true, `migrations.Up` will be applied
+// to the database, potentially making schema changes.
+// If set to false, no database changes will be applied, however the migration version
+// will be checked. If the database is not fully up to date with its migrations, then
+// an error will be returned.
+// nolint:revive // 'migrate' is a control flag.
+func ConnectToPostgres(ctx context.Context, logger slog.Logger, migrate bool, driver string, dbURL string) (sqlDB *sql.DB, err error) {
```

Comment:
Instead of having a boolean control flag, how about `...func(*sql.DB) error`?

This would then become e.g.

```
  ConnectToPostgres(ctx, logger, driver, dbURL, migrations.EnsureClean, migrations.Up)
```

Reply:
I like passing the migration function here, we could always run ensure clean whether or not up is run as well to further simplify usage.

Reply:
I passed in the migration function and always run EnsureClean

Reply:
@Emyrk did you push the change? Not seeing it yet. 🤔